pkg/sloop/store/typed/watchtable_test.go

```diff
@@ -17,11 +16,14 @@ import (
 )
 
 var someTs = time.Date(2019, 1, 2, 3, 4, 5, 6, time.UTC)
+var zeroData time.Time
 
 const someKind = "somekind"
 const someNamespace = "somenamespace"
 const someName = "somename"
-const somePartition = "001546398000"
+const someMinPartition = "001546398000"
+const someMiddlePartition = "001546401600"
+const someMaxPartition = "001546405200"
```

Comment:
these const are shared by all tables(watch,resoureSummary,event,watchActivity),I can move `func (_ *WatchTableKey) SetTestKeys() []string` under these const, but if I move `func (_ *WatchTableKey) SetTestKeys() []string `in this file, it will not make sense, right?

Reply:
I agree that `SetTestKeys` is table specific, but the loop(s) inside it are written to generate N partitions, that control should be moved here and the implementation changed to ensure N partitions are generated.